client/packages/host/src/components/AppDrawer/AppDrawer.tsx

```diff
@@ -151,13 +151,19 @@ export const AppDrawer: React.FC = () => {
   }, [isMediumScreen]);
 
   const onHoverOut = () => {
+    // Hover events not applicable on mobile devices
+    if (EnvUtils.isTouchScreen) return;
     if (!drawer.hoverOpen) return;
 
     drawer.close();
     drawer.setHoverOpen(false);
   };
 
   const onHoverOver = () => {
+    // Hover events not applicable on mobile devices
+    if (EnvUtils.isTouchScreen) return;
+
+    if (EnvUtils.isTouchScreen) return;
```

Comment:
once is probably enough 😉 
```suggestion
    if (EnvUtils.isTouchScreen) return;
```